packages/select/select.scss

```diff
@@ -236,6 +233,7 @@ $select-hover-option-color--inverted: $svaberg;
                 mask-image: $mask-image-url;
                 display: inline-block;
                 margin-left: $component-spacing--xs;
+                margin-top: -#{$component-spacing--xs};
                 width: 17px;
                 height: 13px;
```

Comment:
Denne må ha `flex-shrink: 0;` for ikke å bli skviset av lange alternativer nå som det er `display: flex` :)